src/shipping.ruleset

```diff
@@ -84,6 +84,7 @@
     <Rule Id="SA1639" Action="Hidden" />
     <Rule Id="SA1640" Action="Hidden" />
     <Rule Id="SA1641" Action="Hidden" />
+    <Rule Id="SA1316" Action="None" />
```

Comment:
📝 This rule is configurable in **stylecop.json** (camelCase or PascalCase).

Reply:
Awesome. Because I simply disagree with the default camelCase, since they are exposed as public properties, which generally are supposed to be PaslcalCased.